### PR TITLE
Test_2

### PR DESCRIPTION
**Pull request by svetakanani**
### Evidence :file_folder:

image

### Checklist before requesting a review :ballot_box_with_check:

- [x] I have performed a self-review of my code
- [x] My changes generate no compile time errors

### CODE REVIEW

File: src/sveta/Test_2_StringEvenNo.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package sveta;
+
+public class Test_2_StringEvenNo {
+
+	public static void evenNo(String str) {
+		char ch;
+		int input=0;
+		for(int i=0;i<=str.length()-1;i++) {
```
Comment (MargiVegada) [inline src/sveta/Test_2_StringEvenNo.java:8]: Here, you want last even number, so best approach is to reverse the loop

**Review by MargiVegada — APPROVED** [review]
Where is another program?

File: src/sveta/Test_2_StringEvenNo.java (hunk 1 above)
Comment (svetakanani) [reply src/sveta/Test_2_StringEvenNo.java:8]: ok will do it.

**Comment by svetakanani** [comment]
> Where is another program?

#915  PR no

**Pull request merged**